Sources/Apple/AppleAuthenticator+Models.swift

```diff
@@ -19,9 +19,15 @@ public extension AppleAuthenticator {
     public let userId: String
     public let token: String
     public let authCode: String
-    public let name: String?
```

Comment:
I would still keep the `name` so we don't break the clients who already are using it, and add `PersonNameComponents` as a new property. 
cc: @borut-t?

Reply:
We can then rename the computed property `displayName` to `name`.
And rename `name` to `nameComponents`.
What do you say?

Reply:
sounds good